recipes/simple-websocket-server/all/conanfile.py

```diff
@@ -0,0 +1,56 @@
+from conans import ConanFile, tools, CMake
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class SimpleWebSocketServerConan(ConanFile):
+    name = "simple-websocket-server"
+    homepage = "https://gitlab.com/eidheim/Simple-WebSocket-Server"
+    description = "A very simple, fast, multithreaded, platform independent WebSocket (WS) and WebSocket Secure (WSS) server and client library."
+    topics = ("websocket", "socket", "server", "client", "header-only")
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "compiler", "arch", "build_type"
+    no_copy_source = True
+    generators= "cmake"
+    license = "MIT"
+    requires = (
+        "openssl/1.1.1d",
+    )
+    options = {
+        "use_asio_standalone": [True, False],
+    }
+    default_options = {
+        "use_asio_standalone": True,
+    }
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+        
+    def requirements(self):
+        if self.options.use_asio_standalone:
+            self.requires("asio/1.13.0")
+        else:
+            raise ConanInvalidConfiguration("Is not supported yet. Required boost/1.54 is not available.")
```

Comment:
can you use a newer boost version? like 1.71 or 1.70

Reply:
Oh, my bad, i had another error while creating the package with the build step (removing the build from the package solved it) and i forgot about that option.

Reply:
I tested with boost/1.71.0 too.